apps/metricstream/template.yaml

```diff
@@ -61,11 +62,37 @@ Parameters:
     Description: |
       Buffer incoming data to the specified size, in MiBs, before delivering it
       to the destination.
-
+  ObserveAccountID:
+    Type: String
+    Description: Observe Account Number
+    AllowedPattern: \d*
+  Timestamp:
+    Type: String
+    Description: Timestamp for the metric stream.
+    Default: ''
+    AllowedPattern: "^[0-9]*$"
+  DatasourceID:
+    Type: String
+    Description: >-
+      The datastream for this metric stream.
+    AllowedPattern: '\d*'
+  DomainName:
```

Comment:
nit: I'd call this ObserveDomainName and put it under the `ObserveAcountID` param